CoroutineTemplate/app/src/main/java/co/nimblehq/coroutine/ui/screens/compose/theme/Color.kt

```diff
@@ -4,6 +4,6 @@ import androidx.compose.ui.graphics.Color
 
 @Suppress("MagicNumber")
 object Color {
-    val BlueFreeSpeech = Color(0xFF3F51B5)
-    val AlmostWhite = Color(0xFFDFE4EA)
+    val GreenCitrus = Color(0xFF99CC00)
+    val GreenChristi = Color(0xFF669900)
```

Comment:
Just curious, is it intentional to capitalize the first letter for a color property like this? Shouldn't we do `lowerCamelCase` for properties as per our [guide](https://nimblehq.co/compass/development/code-conventions/kotlin/#properties)?

Reply:
We're following the [official convention](https://developer.android.com/jetpack/compose/themes/material#color) from Android, currently we don't have any conventions for Compose 🙏

Reply:
@minhnimble @Tuubz as far as I know, the Google team explains that the `composable` function, `Color` or `Dimen` variables look like a `factory`  function/ variables to them, so they apply `CamelCase` convention instead of `lowerCamelCase` convention. 💅

Reply:
@Tuubz @minhnimble - indeed, we did not have a Compose convention yet, the initiative is set to be started by @hoangnguyen92dn & @Wadeewee  from their first engineering showcase but has yet to be started. In the meantime, you can refer to this documentation: https://github.com/androidx/androidx/blob/androidx-main/compose/docs/compose-api-guidelines.md as it is written by the Jetpack Compose team themselves. 

Your contribution to the upcoming convention document is also welcome 🙏

Reply:
similar concern showcase: https://github.com/androidx/androidx/blob/androidx-main/compose/docs/compose-api-guidelines.md#dont-1

Reply:
it is time for adding/updating the conventions for support Compose I guess 🚀